web/static/js/components/idea_controls.jsx

```diff
@@ -20,14 +21,15 @@ const IdeaControls = props => {
     ban: isHighlighted,
   })
   const highlightTitle = isHighlighted ? "De-Highlight Idea for Participants" : "Announce Idea to Channel"
+  const canVote = userVoteCounter[currentUser.id] < 5
 
   function renderIcons() {
     if (stage !== "idea-generation" && category !== "action-item") {
       return (
         <VoteCounter
           retroChannel={retroChannel}
           idea={idea}
-          buttonDisabled={stage !== "voting"}
+          buttonDisabled={stage !== "voting" || !canVote}
```

Comment:
rather than a bang here, let's make the variable up above more expressive, as it's only used in this one place and it's being inverted.